src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -277,6 +278,18 @@ private static void EnsureSettingsNode(XmlDocument settings, TestRunSettings set
 
             return null;
         }
+        private static Assembly LoadTestPlatformAssembly()
```

Comment:
github does not allow me comment on the right line here but does the version at ln 333 need an update too?

Reply:
Thanks for catching that, updated